_maps/map_files/Aetherwhisp/Aetherwhisp2.dmm

```diff
@@ -10478,8 +10479,12 @@
 /turf/open/floor/plating,
 /area/maintenance/department/science/central)
 "hbD" = (
-/obj/item/wallframe/airalarm,
-/obj/item/electronics/airalarm,
+/obj/structure/disposalpipe/segment{
+	dir = 4
+	},
+/obj/machinery/airalarm/all_access{
+	pixel_y = 25
+	},
```

Comment:
Ditto from last code review

Reply:
After fighting the variables and just plain breaking the air alarm, I have elected to remove the atmospherics from that hangar entirely.